docs/modules/ROOT/pages/how-to-guides/proc_managed_services_onboarding.adoc

```diff
@@ -3,101 +3,72 @@ When an application is ready for release, the Development team contacts the Mana
 
-* Create a release_strategy.yaml object by running:
+. Create a `ReleasePlanAdmission.yaml` object locally.
 
 +
-[source,]
-----
-oc apply -f releasestrategy.yaml -n managed
-----
-
-+ 
-*Example release_strategy.yaml object*
+*Example `ReleasePlanAdmission.yaml` object*
 
 +
 [source,yaml]
 ----
 apiVersion: appstudio.redhat.com/v1alpha1
-kind: ReleaseStrategy <.>
+kind: ReleasePlanAdmission
 metadata:
-  name: strategy <.>
-  namespace: managed-environment <.>
+ name: sre-production <.>
+ namespace: managed-workspace <.>
 spec:
-  pipeline: release <.>
-  bundle:     quay.io/{ProductName}-release/release-pipeline:main <.>
-  policy: policy <.>
-----
-<.> The name of the resource that the Managed Environment team creates.
-<.> The name of the release strategy.
-<.> The name of the managed environment to which the system deploys an application.
-<.> The name of the pipeline that the system uses when releasing an application to a managed environment.
-<.> The URL to the release pipeline bundle.
-<.> The URL to the enterprise contract policy config file which  stores the policies and collections. The system validates the enterprise contract policies before releasing an application to production.
-
-== The ReleasePlanAdmission object
-
-.*Prerequisites*
-* An existing Development and Managed workspace.
-
-* An existing `ReleasePlan` object in the Development workspace.
-
-* An existing `ReleaseStrategy` object.
-
-.*Procedures*
-
-
-* Create a `release_plan_admission.yaml` object by running:
+ applications:
+  - demo-app <.>
+ data: <key> <.>
+ environment: <sre-production> <.>
+ origin: <dev-workspace> <.>
+ pipelineRef: <pipeline_ref> <.>
+ policy: <policy> <.>
+ serviceAccount: <service-account> <.>
 
-+
-[source,]
-----
-oc apply -f releaseplanadmission.yaml -n managed
 ----
 
 +
-NOTE: The `release_plan_admission.yaml` represents the reciprocal link to the release_plan.yaml objects created by the development team.
+<.> The name of the release plan admission.
+<.> The Managed environment team's workspace.
+<.> A list of applications that you want to enable to be deployed in the managed workspace.
+<.> Optional: An unstructured key used for providing data for the managed Pipeline.
+<.> The environment from which the application updates are allowed to be received in the Managed workspace. This environment is created by the Development team.
+<.> The development team workspace where the application is defined.
+<.> Reference to the Pipeline to be executed by the release service.
+<.> The enterprise contract policy against which the system validates an application before releasing it to production.
+<.> Optional: The name of the service account to use in the Pipeline to gain elevated privileges. It's used only if you have defined the `pipelineRef` value.
 
 +
-*Example `release_plan_admission.yaml` object*
+NOTE: The  ReleasePlanAdmission.yaml represents the reciprocal link to the release_plan.yaml objects created by the development team.
```

Comment:
The other file describes the releaseplan with ReleasePlan.yaml, not release_plan.yaml